app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -227,6 +241,19 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->nullable()
             ->build();
 
+        /*
+         * Modified by Eric J. 6/10/2017.
+         */
```

Comment:
This comment is probably not needed. The authorship and date is logged in the git history.